sql-cli/sql_cli/dag_generator.py

```diff
@@ -83,6 +88,45 @@ def sorted_sql_files(self) -> list[SqlFile]:
 
         return list(depth_first_search.dfs_postorder_nodes(graph))
 
+    def to_transform_task_list(self) -> list[TransformOperator]:
+        """
+        Converts the list of SQL Files into a list of TranformOperator tasks
+        that all have proper dependencies set.
+        :return:
+        """
+        param_dict = {s.path.stem: s.to_transform_operator() for s in self.sql_files}
+        for s in self.sql_files:
+            for p in s.get_parameters():
+                param_dict[s.path.stem].parameters[p] = param_dict[p].output
+                param_dict[s.path.stem].set_upstream(param_dict[p])
+        return list(param_dict.values())
+
+
+def render_dag(directory: Path, workflow_name: str, start_date: datetime = datetime(2020, 1, 1)) -> DAG:
```

Comment:
```suggestion
def get_virtual_dag(directory: Path, workflow_name: str, start_date: datetime = DEFAULT_DATETIME) -> DAG:
```
Suggestions:
* Rename `render_dag` to `get_virtual_dag`
* Have a constant to define the DEFAULT_DATETIME